src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html

```diff
@@ -231,27 +198,9 @@ <h4 *ngIf="displaySubsection(0, 4)">Setting Up Sessions</h4>
       panel is located at the bottom of the Edit Feedback Session page.
     </p>
     <tm-example-box *ngIf="questionsToCollapsed[SessionsSectionQuestions.SESSION_PREVIEW]" @collapseAnim>
-      <div class="card">
-        <div class="card-body background-color-light-blue text-center">
-          <div class="row align-items-center">
-            <div class="col-2 text-right">
-              <b>Preview Session:</b>
-            </div>
-            <div class="col-5">
-              <select class="form-control preview-select">
-                <option *ngFor="let student of exampleStudents" [ngValue]="student.email">[{{ student.teamName }}] {{ student.name }}</option>
-              </select>
-              <button class="btn btn-primary" [disabled]="true" >Preview as Student</button>
-            </div>
-            <div class="col-5" ngbTooltip="View how this session would look like to an instructor who is submitting feedback. Only instructors with submit privileges are included in the list.">
-              <select class="form-control preview-select">
-                <option *ngFor="let instructor of exampleInstructors" [ngValue]="instructor.email">{{ instructor.name }}</option>
-              </select>
-              <button class="btn btn-primary" [disabled]="true" >Preview as Instructor</button>
-            </div>
-          </div>
-        </div>
-      </div>
+      <tm-preview-session-panel [studentsOfCourse]="exampleStudents" [instructorsCanBePreviewedAs]="exampleInstructors"
```

Comment:
Before:
<img width="921" alt="Screenshot 2021-11-06 at 6 48 57 PM" src="https://user-images.githubusercontent.com/7261051/140606957-412a0e09-8427-40ea-996d-5156a5828081.png">
After:
<img width="923" alt="Screenshot 2021-11-06 at 6 48 45 PM" src="https://user-images.githubusercontent.com/7261051/140606956-daac8999-3ab8-4c40-8b13-ca3b18268c32.png">
Although minor, can add `[emailOfStudentToPreview]` and `[emailOfInstructorToPreview]` as well to make a better illustrative example here.

Reply:
fixed in 5d75cbf